examples/complete.yaml

```diff
@@ -13,7 +13,20 @@ spec:
   level: pod # level the disruption should be injected at (can be either pod or node, defaults to pod)
   selector: # label selector[s] to target pods
     app: demo
-    team: developers
+    team: developer
+  advancedSelector: # advanced selectors can select targets on something else than an exact key/value match
+    - key: app
+      operator: Exists
+  # - key: app
+  #   operator: DoesNotExists
```

Comment:
Should this be pluralized? It is not in the PR description. Do both options work?

Reply:
Okay I looked in the code, this is a typo in both examples

Reply:
Oops indeed, thanks! Copy/paste went bad.